NullPointer exception on receiving intent

Related to, but not the same as #981 

Crash occurs during network connectivity change:

```
org.mozilla.mozstumbler.client.mapview.MapActivity.mapNetworkConnectionChanged(MapActivity.java:343)
```

https://github.com/mozilla/MozStumbler/blob/v0.40.1.0/android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapActivity.java#L343

```
java.lang.RuntimeException: Error receiving broadcast Intent { act=android.net.conn.CONNECTIVITY_CHANGE flg=0x10000010 (has extras) } in org.mozilla.mozstumbler.client.mapview.MapActivity$5@40dfaa98
at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:794)
at android.os.Handler.handleCallback(Handler.java:608)
at android.os.Handler.dispatchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:156)
at android.app.ActivityThread.main(ActivityThread.java:4987)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:784)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:551)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.lang.NullPointerException
at org.mozilla.mozstumbler.client.mapview.MapActivity.mapNetworkConnectionChanged(MapActivity.java:343)
at org.mozilla.mozstumbler.client.mapview.MapActivity.access$900(MapActivity.java:62)
at org.mozilla.mozstumbler.client.mapview.MapActivity$5.onReceive(MapActivity.java:357)
at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:781)
... 9 more
java.lang.NullPointerException
at org.mozilla.mozstumbler.client.mapview.MapActivity.mapNetworkConnectionChanged(MapActivity.java:343)
at org.mozilla.mozstumbler.client.mapview.MapActivity.access$900(MapActivity.java:62)
at org.mozilla.mozstumbler.client.mapview.MapActivity$5.onReceive(MapActivity.java:357)
at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:781)
at android.os.Handler.handleCallback(Handler.java:608)
at android.os.Handler.dispatchMessage(Handler.java:92)
at android.os.Looper.loop(Looper.java:156)
at android.app.ActivityThread.main(ActivityThread.java:4987)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:784)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:551)
at dalvik.system.NativeStart.main(Native Method)
```

--- a/android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapActivity.java
+++ b/android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapActivity.java
@@ -340,7 +340,7 @@ public final class MapActivity extends android.support.v4.app.Fragment
 
         final ConnectivityManager cm = (ConnectivityManager) getActivity().getSystemService(Context.CONNECTIVITY_SERVICE);
         final NetworkInfo info = cm.getActiveNetworkInfo();
-        final boolean hasNetwork = (info != null) && cm.getActiveNetworkInfo().isConnected();
+        final boolean hasNetwork = (info != null) && info.isConnected();
         final boolean hasWifi = (info != null) && (info.getType() == ConnectivityManager.TYPE_WIFI);
 
         if (!hasNetwork) {

